frontend/components/common/BaseValueInput.vue

```diff
@@ -38,13 +41,17 @@ export default Vue.extend({
             default: undefined,
         },
         validator: {
-            type: Function as Vue.PropType<function>,
+            type: Function as Vue.PropType<Function>,
```

Comment:
Why this change was needed? `Function` is a js constructor, `function` is a ts type

Reply:
Good catch! This was left over from before and I did not notice it!